<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Eingabe_Net</class>
 <widget class="QDialog" name="Eingabe_Net">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string extracomment="Daten Netzwerk">Data network</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string extracomment="Benutzername:">Username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string extracomment="Kennwort:">Password:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chk_datesave">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Username and password are stored.</string>
   </property>
   <property name="text">
    <string extracomment="Daten speichern">Save data</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_key">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_user">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_go">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string extracomment="Weiter">Next</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_net">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>301</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>15</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string extracomment="Im Netz erreichbare Rechner:">Achievable network computer:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_net">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>320</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>n</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chk_password">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string extracomment="Kennwort anzeigen">Show Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_end">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string extracomment="Abbrechen">Cancel</string>
   </property>
  </widget>
  <zorder>pushButton_net</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>chk_datesave</zorder>
  <zorder>txt_key</zorder>
  <zorder>txt_user</zorder>
  <zorder>pushButton_go</zorder>
  <zorder>listWidget_net</zorder>
  <zorder>label</zorder>
  <zorder>chk_password</zorder>
  <zorder>pushButton_end</zorder>
 </widget>
 <tabstops>
  <tabstop>listWidget_net</tabstop>
  <tabstop>txt_user</tabstop>
  <tabstop>txt_key</tabstop>
  <tabstop>pushButton_go</tabstop>
  <tabstop>pushButton_end</tabstop>
  <tabstop>pushButton_net</tabstop>
  <tabstop>chk_password</tabstop>
  <tabstop>chk_datesave</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>listWidget_net</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>pushButton_net</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
